docs/how_to/create_a_quantum_circuit.rst

```diff
@@ -0,0 +1,169 @@
+
+########################
+Create a quantum circuit
+########################
+
+This guide shows you how to create a quantum circuit.
+
+There are two ways to create a :class:`~.QuantumCircuit` object:
+
+1. Initialize :class:`~.QuantumCircuit` by directly specifying the number of qubits and classical bits you want.
+2. Creating :class:`~.QuantumRegister`\ s and :class:`~.ClassicalRegister`\ s and use the registers to initialize a :class:`~.QuantumCircuit`
+
+.. _create circuit with qubit numbers:
+
+Create by specifying the number of qubits and classical bits
```

Comment:
this feels a bit weird grammatically. I'd suggest saying something like "Create a circuit by specifying..."